common/output-model.cpp

```diff
@@ -81,7 +81,8 @@ void output_model::thread_loop()
                         to_string() << "Failed to fetch firmware logs: " << ex.what());
                 }
             }
-        std::this_thread::sleep_for(std::chrono::milliseconds(10));
+        // FW Limitation
```

Comment:
Please be a little bit more specific what the limit is

Reply:
These are the guidelines to avoid control queues overload and prevent potential issues in FW.
I also documented it in code